public/templates/nav.html

```diff
@@ -0,0 +1,7 @@
+<h1>asdf</h1>
+<ul>
+  <li ng-repeat="nav in navCtrl.navs">
+    <a href="{{nav.url}}">{{nav.title}}</a>
```

Comment:
FYI- Best practice is to use `ng-href` here.

https://docs.angularjs.org/api/ng/directive/ngHref